models/product-variant/src/transformers.ts

```diff
@@ -0,0 +1,30 @@
+import { TAttributeGraphql } from '@commercetools-test-data/attribute';
+import { Transformer } from '@commercetools-test-data/core';
+import type { TProductVariant, TProductVariantGraphql } from './types';
+
+const transformers = {
+  default: Transformer<TProductVariant, TProductVariant>('default', {
+    buildFields: ['price', 'prices', 'images', 'attributes'],
+  }),
+  rest: Transformer<TProductVariant, TProductVariant>('rest', {
+    buildFields: ['price', 'prices', 'images', 'attributes'],
+  }),
+  graphql: Transformer<TProductVariant, TProductVariantGraphql>('graphql', {
+    buildFields: ['price', 'prices', 'images', 'attributes'],
+    replaceFields: ({
+      fields: {
+        attributes,
+        isMatchingVariant,
+        scopedPrice,
+        scopedPriceDiscounted,
+        ...rest
+      },
+    }) => ({
+      ...rest,
+      attributesRaw: attributes as Array<TAttributeGraphql>,
```

Comment:
That `core` change makes it much easier to use the prebuilt fields we define in `buildFields`!

Reply:
Nice!